src/storage/store/node_table.cpp

```diff
@@ -49,7 +47,7 @@ node_offset_t NodeTable::addNodeAndResetProperties(ValueVector* primaryKeyVector
                            primaryKeyVector->getValue<ku_string_t>(pkValPos).getAsString();
         throw RuntimeException(Exception::getExistedPKExceptionMsg(pkStr));
     }
-    for (auto& column : propertyColumns) {
+    for (auto& [propertyID, column] : propertyColumns) {
```

Comment:
`auto& [_, column]` if you don't need propertyID

Reply:
changed `propertyID`->`_`